### ISSUE TITLE
[scrollycoding] Preserve files from previous steps

### ISSUE THREAD

**Issue by pomber** [code-hike/codehike#22]
Instead of repeating the same file for each step:

````md
<Hike>

<StepHead>

```js somefile.js
console.log(1)
console.log(2)
```

</StepHead>

Markdown for step 1

<StepHead focus="1">

```js somefile.js
console.log(1)
console.log(2)
```

</StepHead>

Markdown for step 2

</Hike>
````

Use the files defined in the previous step and only override the ones defined in the current step.

````md
<Hike>

<StepHead>

```js somefile.js
console.log(1)
console.log(2)
```

</StepHead>

Markdown for step 1

<StepHead focus="1" />

Markdown for step 2

</Hike>
````

**Comment by pomber** [comment]
Fixed in 91fcf3ea9ab276da1f0ce1c734b6f25fa216d16c

**Comment by rachelnabors** [comment]
This is great!
